### ISSUE TITLE
Is it possibile add multiple figma files / bins to theemo.js?

### ISSUE THREAD

**Issue by aesign** [gossi/theemo#609]
I'm using the Figma plugin to references color styles and I've got two published "Styles" files: "Styles - Light" and "Styles - Dark". I switch between color modes using the "Swap Library" feature and almost everything works flawlessly.

I've been exporting and converting tokens with a 100% custom script, and I recently wanted to go deep with Theemo's CLI, but I can't figure out how to link those files (and their respective JSON bins) into `theemo.js` though. Is it actually possible?

`  sync: {
    reader: {
      tool: 'figma',
      figmaFile: FIGMA_FILE,
      figmaSecret: FIGMA_SECRET,
      referencer: {
        type: 'figma-plugin',
        plugin: 'theemo',
        pluginConfig: {
          jsonbinFile: JSONBIN_FILE,
          jsonbinSecret: JSONBIN_SECRET,
          formats: {
            color: 'hex',
            colorAlpha: 'rgb'
          }
        }
      },`

**Comment by gossi** [comment]
Yeah, I've recently realized, this would be quite good to aggregate tokens from multiple source files. Currently, this is not possible with a config.... but I think, a little workaround can do the job here. You have to call `theemo sync` twice (one for each file):

```json
"scripts": {
  "sync:light": "FIGMA_FILE=... JSONBIN_FILE=... theemo sync",
  "sync:dark": "FIGMA_FILE=... JSONBIN_FILE=... theemo sync"
}
```

That's how I'd put this in my `package.json` and a `sync` script to execute them both. In `.env` you only keep the secrets and prepend the relevant environment variables per script.

**Comment by gossi** [comment]
After I reworked Theemo... now it is possible =)

Documentation on how to do so:

- https://theemo.io/sync
- https://theemo.io/sync/reader

I can happily close this one now 😃
